### PR TITLE
chore: update ui-kit to v6

### PR DESCRIPTION
**Pull request by montezume**
Update UI-Kit to [v6.0.0](https://www.npmjs.com/package/@commercetools-frontend/ui-kit/v/6.0.0)

* Delete PasswordField
* Fix tokens import
* Moved SubHeadline to use h4 instead of h3 (as h3 isn't supposed to be an option for that)

### CODE REVIEW

File: packages/application-shell/src/components/login-sso/__snapshots__/login-sso.spec.js.snap [hunk 1]
```diff
@@ -9,7 +9,7 @@ exports[`rendering <Formik> form elements should match layout 1`] = `
     scale="m"
   >
     <TextSubheadline
-      elementType="h3"
+      elementType="h4"
```
Comment (montezume) [inline packages/application-shell/src/components/login-sso/__snapshots__/login-sso.spec.js.snap]: h3 isn't supposed to be part of subheadline

**Comment by montezume** [comment]
It's nice to have proptype warnings again 💯

**Comment by tdeekens** [comment]
> It's nice to have proptype warnings again 💯

That was really a bad hiccup :)

**Comment by emmenko** [comment]
We can merge this here, there is no problem with that. We could also do a new release here (but I would wait for that).

**Pull request merged**